client/components/mma/identity/models.ts

```diff
@@ -349,23 +349,17 @@ interface CountryValues {
 }
 export const COUNTRIES: CountryValues[] = [
 	{ iso: 'AF', name: 'Afghanistan' },
-	{ iso: 'AL', name: 'Albania' },
```

Comment:
Would it make more sense to have a list of these and filter them out from this master list of all countries?

Reply:
The code would be clearer with a filter out. But it's more secure / guaranteed if we remove them. Either is fine with me

Reply:
just thought it might be easier if the list changes to keep track of it separately than to eg add a country back in

Reply:
it also might not be clear to someone coming to the codebase why there is an incomplete list of ISO countries

Reply:
We would need to remove or a add the country from the non-servicable list if anything was change. It just depends on how careful we want to be